### PR TITLE
Add new site: adds new selection popover

### PR DESCRIPTION
**Pull request by keoshi**
> This PR is on-hold until we discuss a better way to offer this choice to users.

This PR adds a popover for new site selection.

Addresses https://github.com/Automattic/wp-calypso/issues/22315

Related: https://github.com/Automattic/wp-calypso/pull/23093

image

### CODE REVIEW

File: client/components/site-selector/add-site.jsx [hunk 1]
```diff
@@ -10,23 +10,90 @@ import Gridicon from 'gridicons';
...
+					context={ this.popoverToggle }
+				>
+					<PopoverMenuItem
+						href={ this.getNewWpcomSiteUrl() }
+						onClick={ this.recordPopoverAddNewSite }
+					>
+						<span>
+							<span>{ translate( 'Create a new site' ) }</span>
+							<span className="site-selector__popover-item-details">
+								{ translate( 'Start a new WordPress.com site' ) }
```
Comment (a8ci18n) [inline client/components/site-selector/add-site.jsx]: Hi! I've found a possible matching string that has already been translated **26** times:
`translate( 'Start a new site on WordPress.com' )` *ES Score: 12*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

Comment (keoshi) [reply client/components/site-selector/add-site.jsx]: Thanks, bot! 🤖

**Comment by joanrho** [comment]
Works well! What do you think about adding a light horizontal rule between the two options?

**Comment by keoshi** [comment]
@joanrho updated the PR!

**Comment by joanrho** [comment]
Looks great! 

minor nitpick: can you make the spacing above and below the text the same? Might need to nudge down the margin (top/bottom) on the separator to like ~4px-ish.

**Comment by joanrho** [comment]
Thanks for making that adjustment. Just tested again and LGTM!

**Comment by alisterscott** [comment]
Is this PR still required?

**Comment by keoshi** [comment]
Yeah, this has been put on hold while we work on higher priority, but this PR is still valid.

**Comment by millerf** [comment]
@gwwar  I think this one can be deleted as the user flow has changed and  we have the selector directly on the next page...
Screenshot 2020-05-04 at 19 32 22

**Comment by gwwar** [comment]
@keoshi I'm going to close this out since it's 2 years old :D

**Comment by keoshi** [comment]
@millerf @gwwar We already had that view, but we got some feedback that it was confusing, hence the solution in this PR. Since a decision was never made, I'm totally ok will leaving this closed.

**Pull request closed without merging**